src/Microsoft.Data.Analysis/StringDataFrameColumn.cs

```diff
@@ -169,12 +170,36 @@ public IEnumerator<string> GetEnumerator()
 
         public override DataFrameColumn Filter<U>(U lower, U upper) => throw new NotSupportedException();
 
+        public new StringDataFrameColumnRollingWindow Rolling(int windowSize)
```

Comment:
You should also be overriding the base `Rolling` method.

Reply:
Done.